internal/spentcalories/spentCalories.go

```diff
@@ -41,42 +56,48 @@ func meanSpeed(steps int, duration time.Duration) float64 {
 // data string - строка с данными.
 // weight, height float64 — вес и рост пользователя.
 func TrainingInfo(data string, weight, height float64) string {
-	// ваш код ниже
+	steps, trainingType, trainingTime, _ := parseTraining(data)
+	var calories float64
+	duration := trainingTime.Hours()
+	distance := distance(steps)
+	speed := meanSpeed(steps, trainingTime)
+
+	switch trainingType {
+	case "Бег", "бег":
+		calories = RunningSpentCalories(steps, weight, trainingTime)
+
+	case "Ходьба", "ходьба":
+		calories = WalkingSpentCalories(steps, weight, height, trainingTime)
+
+	default:
+		trainingType = "неизвестный тип тренировки"
```

Comment:
Да нужно было просто
```suggestion
		return "неизвестный тип тренировки"
```